<launch>
  <group ns="/sensor0">

	<param name = "posx" type = "double" value = "5.02"/>
	<param name = "posy" type = "double" value = "2.1"/>
	<param name = "angle" type = "double" value = "65.78"/> <!-- Field of view -->
	<param name = "senrange" type = "double" value = "1.4"/> 
	<param name = "yaw" type = "double" value = "60.5"/>
  	<param name = "sensortype" type = "double" value = "5"/> <!-- enter 0:radar , 1:camera/Lidar, 5:Ideal sensor -->
	<param name = "rangerr" type = "double" value = "0.12"/> <!-- For radar only -->
	<param name = "velerr" type = "double" value = "0.1"/> <!-- For radar only -->
	<param name = "azierr" type = "double" value = "0.2"/> <!-- For radar only -->
	<param name = "posxerr" type = "double" value = "0.2"/> <!-- For camera/lidar only -->
	<param name = "posyerr" type = "double" value = "0.2"/> <!-- For camera/lidar only -->
	
	<!-- Change below 2 param values prefix to group name -->

	<param name = "name" type = "str" value = "/sensor0/obj_list" /> 
	<param name = "egodataname" type = "str" value = "/sensor0/ego_data" /> 
	




  	<node pkg="sensor_model" type = "sensor_model.py" name="sensor0"   />
	<node name="Sensor2Ego" pkg="sensor_model" type="sensor2Ego.py"   />
	<node name="KF" pkg="sensor_model" type="KF.py" output="screen"  /> 
	
  </group>

  <node name="OSI_ROS_dummy" pkg="object_list" type="OSI_ROS_dummy.py"  />



 
</launch>

